front/app/containers/IdeasEditPage/IdeasEditForm.tsx

```diff
@@ -190,7 +190,6 @@ const IdeasEditForm = ({ ideaId }: Props) => {
       idea_images_attributes,
       location_point_geojson,
       project_id: projectId,
-      publication_status: 'published',
```

Comment:
When a proposal/input had the `prescreening` `idea_status`, this would get sent to the BE, resulting in a 422.